README.md

```diff
@@ -1352,6 +1354,8 @@ Tasks queues receive tasks and their related data, runs them, then delivers thei
 
 If queues start to grow significantly, the queue size can become larger than memory, resulting in cache misses, disk reads, and even slower performance.  [Back pressure](http://mechanical-sympathy.blogspot.com/2012/05/apply-back-pressure-when-overloaded.html) can help by limiting the queue size, thereby maintaining a high throughput rate and good response times for jobs already in the queue.  Once the queue fills up, clients get a server busy or HTTP 503 status code to try again later.  Clients can retry the request at a later time, perhaps with [exponential backoff](https://en.wikipedia.org/wiki/Exponential_backoff).
 
+In Systems like apache kafka we can set up parallel consumers (those needn't acknowledge before another set of message comes to them). Backpressure comes to effect in this scenario as well if you want to hold the proeucer from overwhelming the consumer.
```

Comment:
```In Systems like apache kafka``` -> maybe ```In systems like Apache Kafka``` would be better

```proeucer``` -> ```producer```
This is a typo I guess

Reply:
yeah changed!!